README.md

```diff
@@ -119,7 +122,25 @@ $ make run
 
 ## Access the controlplane
 
-The kubeconfig file of the controlplane is in the dir `hack/deploy/cert-${HUB_NAME}/kubeconfig`.
+- If you run the controlplane as a binary, the controlplane kubeconfig file is in `_output/controlplane/.ocm/cert/kube-aggregator.kubeconfig`
+
+- If you deploy the controlplane in a cluster, run the following command to get the controlplane kubeconfig
+
+  ```bash
+  kubectl -n multicluster-controlplane get secrets multicluster-controlplane-kubeconfig -ojsonpath='{.data.kubeconfig}' | base64 -d > multicluster-controlplane.kubeconfig
+  ```
+
+  If you enable the authentication delegating, you can set a context for your controlplane in your cluster kubeconfig with the following commands
+
+  ```bash
+  external_hostn_ame=<your controplane external host name>
+  # if you want to add the ca of your cluster kube-apiserver, using the command:
+  # kubectl config set-cluster multicluster-controlplane --server="https://${external_host_name}" --embed-certs --certificate-authority=<the ca path of your cluster kube-apiserver>
+  kubectl config set-cluster multicluster-controlplane --server="https://${external_host_name}" --insecure-skip-tls-verify
```

Comment:
can we provide a command to get the openshift ca? if so, we can remove --insecure-skip-tls-verify.

Reply:
for ocm, I think we may let user to specify the ca if user want to have a ca, because we may have more plateform to support